.release-notes/ponybench-rename.md

```diff
@@ -0,0 +1,15 @@
+## Rename `ponybench` package to match standard library naming conventions
+
+We recently realized that when we renamed large portions of the standard library to conform with our naming standards, that we missed the `ponybench` package. To conform with the naming convention, the `ponybench` package as been renamed to `pony_bench`.
+
+You'll need to update your test code from:
+
+```pony
+use "pony_bench"
```

Comment:
This should be `ponybench` here in the "before" code block.

Reply:
Fixed